manifests/v2/third-party/jobset/manifests.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: jobset
+    app.kubernetes.io/instance: system
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/part-of: jobset
+    control-plane: controller-manager
```

Comment:
What are these labels eventually? `Any type of name prefix or suffix are applied to values?

Reply:
Oh, sorry, forgot to remove this file.